src/data/tribers/alin.md

```diff
@@ -0,0 +1,14 @@
+---
+name: 'Alin Chindea'
+words:
+  - Learning
+  - Cycling
+  - Storytelling
+image: images/alin.jpg
+social:
+  twitter: 'alinchindea'
+  github: 'AlinChindea'
+  linkedin: 'alinchindea'
+---
+
+After a dozen years working with the United Nations I shifted my career to programming. I experiment with JavaScript and PHP since 2017 and I strive to be a better developer every day. I joined Neontribe in 2019 hoping to use my skills in tech-for-good. When I don't dream about code (seriously now, when do the JavaScript dreams end???), I think about food, cycling along rivers that cross more than one country and 'untranslatable' words from any language (such as Sisu - Finnish for extraordinary determination/courage, especially in the face of adversity - and Utepils - Norwegian for a beer that is enjoyed outside particularly on the first warm and sunny day of the year). Addicted to stories and storytelling, Coen brothers movies and playing football. I do not have a pet alpaca.
```

Comment:
Maybe it should read 'I have experimented with JavaScript and PHP since 2017' instead of 'I experiment with JavaScript and PHP since 2017'.   Everything else is great! 👍

Reply:
I agree with Holly, I think it would read better like this:

```suggestion
After a dozen years working with the United Nations I shifted my career to programming. I have experimented with JavaScript and PHP since 2017 and I strive to be a better developer every day. I joined Neontribe in 2019 hoping to use my skills in tech-for-good. When I don't dream about code (seriously now, when do the JavaScript dreams end???), I think about food, cycling along rivers that cross more than one country and 'untranslatable' words from any language (such as Sisu - Finnish for extraordinary determination/courage, especially in the face of adversity - and Utepils - Norwegian for a beer that is enjoyed outside particularly on the first warm and sunny day of the year). Addicted to stories and storytelling, Coen brothers movies and playing football. I do not have a pet alpaca.
```

Reply:
thank you!